### DISCUSSION TITLE
如何优雅的实现Modal添加快捷键绑定ok、cannel方法？

### DISCUSSION THREAD

**Discussion by dbssAlan** [ant-design/ant-design#45696] [Q&A]
如题，目前Modal组件只支持Esc这个按键的。但在一些工业生产中，出于节约时间和操作便利性的考虑，有使用方向键判断弹窗对错的需求。请问大家如何优雅的实现这一功能呢？

我自己尝试的一些，需要`useEffect`去绑定监听键盘监听事件，加上使用`Modal.useModal()`方法需要不断暴露`contextHolder`。对于复用，不够友好。目前没有想到一个很优雅的方法。是否需要antd官方支持?

**Comment by vaynevayne** [comment]
https://mantine.dev/hooks/use-hotkeys/
